src/Pages/LeaderBoard.js

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from "react-router-dom";
+import data from "./config.contracts.json"
+
+var protocols = []
+var contractSets = []
+var VaRofContracts = ["$$$","$$$","$$$", "$$$", "$$$", "$$$"]
+
+var navigate;
+
+export var chosenContracts = []
+export var chosenVaRofContracts = []
+export var chosenProtocolName
+
+async function sortData(){
+
+  data.map( (data) => {
+      protocols.push(data.name)
+      var list = JSON.parse(JSON.stringify(data.contracts))
+      contractSets.push(list)
+  })
+
+}
+
+async function navigateToProtolPage(i, name){
+  // chosenVaRofContracts = []
```

Comment:
Commented out section added here for future work. This is relevant for when subgraph is indexed.